### PR TITLE
#5: Added the readme.md file.

### PR DESCRIPTION
**Pull request by ulin-evgeny**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+## Introduction
+
+This plugin adds the ability for users to work with media files.
+
+## Installation
+
+1. Install the package using the following command: `composer require ronasit/laravel-media`
+2. Run `php artisan vendor:publish`
+3. Add `src/MediaServiceProvider` to `config/app.php` your application
```
Comment (DenTray) [inline readme.md:11]: this step requires for the Laravel < 5.5

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+### Configs
+
+- `media.permitted_types` - list of allowed media file types.
+- `media.classes.user_model` - user model class that will have a relationship with the media class.
```
Comment (DenTray) [inline readme.md:16]: Let's remove configs block from the readme and move user configuration step to the installation block

**Review by DenTray — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (ulin-evgeny) [reply readme.md]: Fixed.

File: readme.md (hunk 2 above)
Comment (ulin-evgeny) [reply readme.md]: Done.

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+## Introduction
+
+This plugin adds the ability for users to work with media files.
+
+## Installation
+
+1. Install the package using the following command: `composer require ronasit/laravel-media`
+2. Run `php artisan vendor:publish`
+3. For Laravel <= 5.5 add `RonasIT\Media\MediaServiceProvider::class` to config `app.providers` list.
+4. In `media.permitted_types` config, specify the allowed types of media files.
```
Comment (DenTray) [inline readme.md:12]:
```suggestion
```

File: readme.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+5. In `media.classes.user_model` config, specify the model class that will have a relationship with the media class.
```
Comment (DenTray) [inline readme.md:13]:
```suggestion
4. Set your project's User model to the `media.classes.user_model` config.
```

File: readme.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
```
Comment (DenTray) [inline readme.md:14]:
```suggestion

## Integration with [LaravelSwagger](https://github.com/RonasIT/laravel-swagger)

This package includes OpenAPI documentation file. To include it to your project's documentation, you need to register it in the `auto-doc.additional_paths` config:

`vendor/ronasit/laravel-media/documentation.json`
```

**Review by DenTray — CHANGES_REQUESTED** [review]

File: readme.md (hunk 5 above)
Comment (ulin-evgeny) [reply readme.md:17]: Done.

File: readme.md (hunk 4 above)
Comment (ulin-evgeny) [reply readme.md]: Done.

File: readme.md (hunk 3 above)
Comment (ulin-evgeny) [reply readme.md]: Done.

File: readme.md (hunk 5 above)
Comment (DenTray) [reply readme.md:17]: nope, please check empty lines in the suggestion

**Review by DenTray — CHANGES_REQUESTED** [review]

File: readme.md (hunk 5 above)
Comment (ulin-evgeny) [reply readme.md:19]: Fixed.

**Review by DenTray — APPROVED** [review]

**Pull request merged**